package/README.md

```
@@ -590,6 +595,43 @@ const CustomComponent = () => {
+| `AudioPlayer.Volume` | volume trigger + slider |
+| `AudioPlayer.PlayList` | sortable playlist drawer (accepts `initialExpanded?`) |
+| `AudioPlayer.PlayButton` | Play + Prev + Next group |
+| `AudioPlayer.RepeatButton` | repeat-type button |
+| `AudioPlayer.Artwork` | track artwork |
+| `AudioPlayer.TrackInfo` | track title / writer |
+| `AudioPlayer.TrackTime` | current + duration time |
+| `AudioPlayer.CustomComponent` | user-defined slot |
+
+Each slot accepts `gridArea?` and `visible?` plus its own domain props.
+
+### Mental model — `activeUI` vs compound children
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)